packages/dataviews/src/dataviews.js

```diff
@@ -28,15 +29,34 @@ export default function DataViews( {
 	actions,
 	data,
 	getItemId = defaultGetItemId,
+	getItemTitle,
 	isLoading = false,
 	paginationInfo,
 	supportedLayouts,
 	onSelectionChange = defaultOnSelectionChange,
 	onDetailsChange = null,
 	deferredRendering = false,
+	labels,
```

Comment:
Can we provide default labels, such as `Deselect: (title)` or `Deselect item: (title)` so consumers don't _have_ to?

I'd even go further and don't introduce this property in this PR. We may want to later, but I'd like us to explore alternatives where consumers don't have to provide any labels at all to DataViews. The users already know _where_ they are (templates, pages, patterns) when they experience these labels (in a visual or auditive way). So, generic approaches such as "Deselect item" may work well. If we strongly wanted to do this, I agree a single object `labels` is the cleaner approach.

Reply:
We already provide defaults, if consumers don't provide the labels prop. The labels prop is optional if consumers want to provide better labels. I will follow the feedback and just have the generic version.

Reply:
Ah, I saw the defaults now. What do you think of doing the following?

- The `DataViews` components prepares the `labels` in the same way it prepares the `fields`. Something along these lines (pseudocode):

```javascript
const _labels = useMemo(
  return {
    selectItem: labels.selectItem || __( 'Select a new item' ),
    deselectItem: labels.deselectItem || __( 'Deselect a new item' ),
  };
[ labels ] );

// ... 

<ViewComponent
  labels={ _labels }
/>
```

- The view component access the labels directly, because they are always present. We can remove the conditionals there. It also has the benefit of all views having the same labels.

Reply:
I ended up following your previous feedback and removed the labels property.